test/browser/browser.sh

```diff
@@ -22,7 +22,7 @@ dnf install --disablerepo=fedora-cisco-openh264 -y firefox
 sed -ie 's/#SHA_CRYPT_MAX_ROUNDS 5000/SHA_CRYPT_MAX_ROUNDS 5000/' /etc/login.defs
 
 # HACK: https://bugzilla.redhat.com/show_bug.cgi?id=2057769
-if [ "$(rpm -q edk2-ovmf)" = "edk2-ovmf-20220126gitbb1bba3d77-3.el9.noarch" ]; then
+if [ "$(rpm -q libvirt)" = "libvirt-8.0.0-5.el9.x86_64" ]; then
```

Comment:
```
++ rpm -q libvirt
+ '[' 'package libvirt is not installed' = libvirt-8.0.0-5.el9.x86_64 ']'
```
That's what I meant with "I don't know the exact package name". `libvirt-daemon` should be fine, the `rpm -q` below tests it.

See [log](https://artifacts.dev.testing-farm.io/abf29a0d-108c-4950-9663-7e3a0268bd3f/)

Reply:
Ah, I thought libvirt will be available since it's listed http://mirror.stream.centos.org/9-stream/AppStream/x86_64/os/Packages/ , didn't know our test image doesn't have it